apps/passport-client/src/pcdPackages.ts

```diff
@@ -62,6 +63,9 @@ async function loadPackages(): Promise<PCDPackage[]> {
   await MessagePCDPackage.init?.({});
 
   await PODPCDPackage.init?.({});
+  await GPCPCDPackage.init?.({
+    zkArtifactPath: "/artifacts/test/proto-pod-gpc"
```

Comment:
Is it worth modifying the service worker to pre-cache this artifact in the same way that we do for Semaphore and ZKEdDSAEventTicketPCD artifacts?

Reply:
Yes, though in a more limited way.  Two comments for now:

1) None of these artifacts are available outside of dev mode yet, so it's not useful to tackle the service worker yet.
2) I don't think we can/should pre-cache artifacts for all of the circuits in the family (of which there will eventually be many), but we should pick the specific sizes used for common use cases (notably tickets) and pre-cache those.  When 3rd-party devs are building custom use cases, their users will have to be online to download for first-time use.